### PR TITLE
fix(preview-middleware): set qunit autostart to false

### PR DESCRIPTION
**Pull request by heimwege**
Set  qunit autostart to false as recommended by @matz3 [here](https://github.com/SAP/open-ux-tools/pull/1726/files#r1549565997)

### CODE REVIEW

File: packages/preview-middleware/templates/test/qunit.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 sap.ui.define([
     'sap/ui/thirdparty/qunit-2'
 ], function() {
```
Comment (matz3) [inline packages/preview-middleware/templates/test/qunit.js:3]: I'd suggest to rather access `QUnit` from the sap.ui.define import, not via global.

Comment (heimwege) [reply packages/preview-middleware/templates/test/qunit.js]: oh, you are right 🙈

Comment (matz3) [reply packages/preview-middleware/templates/test/qunit.js]: Looks better now 👍🏻

**Review by matz3 — APPROVED** [review]
LGTM

**Review by tobiasqueck — APPROVED** [review]
* changeset ok
* successfully tested it manually 
* simple change is straight forward and clear nothing else needed

**Review by cianmSAP — APPROVED** [review]
Update looks good
Snapshot updated
Changeset 
✅

**Pull request merged**